modules/core/src/lib/utils/utils.ts

```diff
@@ -34,3 +35,8 @@ export function isObjectEmpty(obj: object): boolean {
   }
   return isEmpty;
 }
+
+
+export function isUniform(value: unknown): boolean {
```

Comment:
- `IsUniformValue` perhaps? 
- Could it be co-located with other uniform related code, rather than in this very generic file? Or at least in its own file here in the same directory.